type: choice
is_multiple_choice: false
options:
- text: 'It means that the exception was raised in the file that was executed. '
- text: <code>module.py</code> is the name of the file where the exception appeared.
- text: It says that the error occurred in another file.
- text: <code>module()</code> is the function where the error was triggered.
files:
- name: main.py
  visible: true
  text: |+
    #  You can experiment here, it wonâ€™t be checked

  learner_created: false
feedback_link: https://hyperskill.org/learn/step/14995#comment
status: Solved
record: -1
local_check: false
